homeassistant/components/alexa/state_report.py

```diff
@@ -85,8 +85,10 @@ async def async_entity_state_listener(
         if not should_report and not should_doorbell:
             return
 
-        if should_doorbell:
-            if new_state.state == STATE_ON:
+        if should_doorbell and new_state:
+            if (
+                old_state is None or old_state.state != STATE_ON
```

Comment:
Could this be simplified? `old_state.state` should probably be `STATE_OFF` which is not `None`. The `or` is not needed I think,